README.md

```diff
@@ -1,147 +1,3 @@
-
-In the Maps App, LocatorTasks are initialized using an online locator provided by an ArcGIS service.
-
-```
-// The Locator Task class variable is initialized using the ArcGIS World Geocoding Server.
-mLocator = new LocatorTask(getString("http://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer");
-```
-
-You can also provision your own [custom geocode service](https://doc.arcgis.com/en/arcgis-online/administer/configure-services.htm#ESRI_SECTION1_0A9A071A7AB748028C8213D1D863FA18) to support your organization. Before using the LocatorTask for geocode or searching for places, the LocatorTask must be LOADED. The loadable pattern is described [here](https://developers.arcgis.com/android/beta/guide/loadable-pattern.htm). LocatorTask operations are performed asynchronously using ListenableFutures, an implementation of Java’s Future interface. ListenableFutures add the ability to attach a listener that runs upon completion of the task. One of the first user interactions the Maps App supports is suggesting places near the device location.
-
-## Place Suggestions
-
-Typing the first few letters of a place into the Map App search box (e.g. “Voodoo Doughnut”) shows a number of suggestions near the device’s location
-
-```
-// Attach a listener to the locator task since the LocatorTask may or may not be loaded the
-// the very first time a user types text into the search box.  If the Locator is already loaded,
-// the following listener is invoked immediately.
-mLocator.addDoneLoadingListener(new Runnable() {
-    @Override 
-    public void run() {
-        final ListenableFuture> suggestionsFuture = mLocator.suggestAsync(query, suggestParams);
-        // Attach a done listener that executes upon completion of the async call
-        suggestionsFuture.addDoneListener(new Runnable() {
-            @Override
-            public void run() {
-                try {
-                    // Get the suggestions returned from the locator task.
-                    // Process and display
-
-                } catch (Exception e) {
-                   dealWithException(e);
-            }
-          }
-        });
-    }
-});
-// Initiate the asynchronous call
-mLocator.loadAsync();
-```
-
-## Features
-* Dynamically switch basemaps
-* Place search
-* Routing
-* Geocode addresses
-* Reverse geocode
-* Sign in to ArcGIS account
 
 ## Development Instructions
```

Comment:
Add an into above this for context

# Overview
The maps-app is your organizations mapping application that allows for switching basemaps, searching, routing, and authenticating against your ArcGIS organization for custom basemaps.